x-pack/plugins/security_solution/public/overview/components/overview_empty/index.test.tsx

```diff
@@ -57,7 +61,7 @@ describe('OverviewEmpty', () => {
           fill: false,
           label: 'Add Endpoint Security',
           onClick: undefined,
-          url: '/app/home#/tutorial_directory/security',
+          url: `#/integrations/endpoint-${endpointPackageVersion}/add-integration`,
```

Comment:
FYI:
I had suggested somewhere (can't find it now) that the Fleet integration page support "keywords" in the UI url instead of version, so that we did not have to pull this information before building a link. The idea as to support something like: `#/integrations/endpoint-installed/add-integration` or `#/integrations/endpoint-latest/add-integration` and the integration page would just "do the right thing" in pulling in the package information based on the keywords `installed` or `latest`.  But that has not had any movement.
We also have this issue opened in Fleet to provide a better API for retrieving the package version number - https://github.com/elastic/kibana/issues/66777

cc/ @jen-huang , @jfsiii

Reply:
that would be very cool.

Reply:
not exactly the same as keywords, but here is the enhancement issue for allowing to link without version: https://github.com/elastic/kibana/issues/93393
it would direct to either the currently installed version, or the latest registry version if not installed
I think we should have this implemented in this release :)